extensions/ql-vscode/src/queries-panel/query-pack-discovery.ts

```diff
@@ -37,7 +37,7 @@ export class QueryPackDiscovery extends FilePathDiscovery<QueryPack> {
    */
   public getLanguageForQueryFile(queryPath: string): QueryLanguage | undefined {
     // Find all packs in a higher directory than the query
-    const packs = this.getPathData().filter((queryPack) =>
+    const packs = (this.getPathData() || []).filter((queryPack) =>
```

Comment:
It's a bit confusing for me to run a filter over an empty array, but checking `getPathData` first and having another if condition is not really better I suppose?